Containers/talk/Dockerfile

```diff
@@ -67,5 +67,5 @@ USER talk
 ENTRYPOINT ["start.sh"]
 CMD ["/usr/bin/supervisord", "-c", "/supervisord.conf"]
 
-HEALTHCHECK CMD (nc -z localhost 8081 && nc -z localhost 8188 && nc -z localhost 4222 && nc -z localhost $TALK_PORT) || exit 1
+HEALTHCHECK CMD (nc -z localhost 8081 && nc -z localhost 8188 && nc -z localhost 4222 && nc -z localhost "$TALK_PORT" && nc -z "$NC_DOMAIN" "$TALK_PORT") || exit 1
```

Comment:
I fear this will not work behin cloudflare so this would mean it would not be possible to fix the healthcheck behind that?

Reply:
It would mark the talk container as unhealthy, if the container runs behind cloudflare or the firewall is closed

Reply:
> It would mark the talk container as unhealthy, if the container runs behind cloudflare

yes, but this is not what we want, no?

> the firewall is closed

That would actually be an improvement, yes

Reply:
> > It would mark the talk container as unhealthy, if the container runs behind cloudflare
> 
> yes, but this is not what we want, no?
why? If talk runs behind cloudflare, the talk container would never work.. so it would be unhealthy

Reply:
True but in that case they should then spin-up their own turn server and hpb and keep the talk container enabled (because otherwise the spreed app gets disabled) which would then always report it as unhealthy

Reply:
AIO disabled spread, if talk is disabled?

Reply:
yes, if you disable the talk option in the aio interface, the spreed app also gets disabled...

Reply:
but why this? would it not be better to let the app enabled? Because I think this change would be a good improvement to the healthcheck of the talk container

Reply:
It is the same logic for the other apps as well since we want to make sure that removing the talk container app does not break the app. If the talk container would be disabled and the spreed app kept enabled it would basically break it because the turnserver and hpb would of course not work...

Reply:
and if the app only get disabled if one of hpb/stun/turn settings is set to the aio defaults?

Reply:
> and if the app only get disabled if one of hpb/stun/turn settings is set to the aio defaults?

I fear this would be much more complicated and also more error-prone...

Reply:
> True but in that case they should then spin-up their own turn server and hpb and keep the talk container enabled (because otherwise the spreed app gets disabled) which would then always report it as unhealthy

but where is there the problem, if the container is  not reachable, it is unhealthy, even if the people don't use it...

Reply:
okay, lets do this then